browser/src/NeovimInstance.ts

```diff
@@ -126,6 +126,14 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
                             this.emit("event", eventName, eventContext)
                         } else if (pluginMethod === "window_display_update") {
                             this.emit("window-display-update", args[0][0], args[0][1])
+                        } else if (pluginMethod === "api_info") {
+                            const apiVersion = args[0][0]
+                            if (apiVersion.minor < 1) {
```

Comment:
Only the API level should be checked (that's the entire purpose of having an API level). The nvim major/minor versions are irrelevant regarding API compatibility.